src/utils/createFiltersObjFromSearchParams.ts

```diff
@@ -0,0 +1,20 @@
+﻿import { JobOfferFiltersEnum } from '@/app/(profile)/types'
+import { type ReadonlyURLSearchParams } from 'next/navigation'
+
+export const createFiltersObjFromSearchParams = (
+  searchParams: ReadonlyURLSearchParams,
```

Comment:
**nit:** This is filter related thing - it should be places somewhere in `app/(profile)/types`. We should try to not import anything from `app/(domain)` to the root directories